app/src/main/res/values/integers.xml

```diff
@@ -2,4 +2,10 @@
 <resources>
     <integer name="offer_name_max_size">50</integer>
     <integer name="offer_description_max_size">1000</integer>
+    <integer name="username_min_length">1</integer>
+    <integer name="username_max_length">25</integer>
+    <integer name="user_infos_min_length">0</integer>
+    <integer name="user_infos_max_length">200</integer>
+    <integer name="user_preferences_min_length">0</integer>
+    <integer name="user_preferences_max_length">100</integer>
```

Comment:
So you still needed to add these in Java and XML? If we change our mind about the value, we will have to update both places?

Reply:
These are not used anymore. I deleted these so there will be no confusion which to use in the future.